lib/core/src/model.rs

```diff
@@ -183,6 +183,23 @@ pub struct ReceivePaymentResponse {
     pub invoice: String,
 }
 
+#[derive(Debug, Serialize)]
+pub struct OnchainPaymentLimitsResponse {
+    /// Minimum swap amount for a Send Payment to be valid
+    pub send_min_amount_sat: u64,
+    /// Maximum swap amount for a Send Payment to be valid
+    pub send_max_amount_sat: u64,
+    /// Maximum swap amount which the swapper will accept for zero-conf Send Payments
+    pub send_max_amount_sat_zero_conf: u64,
```

Comment:
I think it reads better like this, also with the unit at the end
```suggestion
    pub send_max_zero_conf_amount_sat: u64,
```

Reply:
Done